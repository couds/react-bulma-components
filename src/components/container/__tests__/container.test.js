import React from 'react';
import { render } from 'react-testing-library';
import { Container } from '..';

describe('Container component', () => {
  it('should have container classname', () => {
    const { asFragment } = render(
      <Container>
        <div className="bd-notification is-success">
          <p>Default</p>
          <p>Container</p>
        </div>
      </Container>
    );
    expect(asFragment()).toMatchSnapshot();
  });

  it('should have helper classnames', () => {
    const { asFragment } = render(
      <Container
        paddingless
        responsive={{
          widescreen: {
            display: {
              value: 'block',
              only: true
            }
          },
          touch: {
            display: {
              value: 'flex'
            }
          },
          desktop: {
            hide: {
              value: true
            }
          }
        }}
      >
        <div className="bd-notification is-success">
          <p>Default</p>
          <p>Container</p>
        </div>
      </Container>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
