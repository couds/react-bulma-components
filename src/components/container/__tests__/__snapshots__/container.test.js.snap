// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component Should have container classname 1`] = `
<div
  className="container"
  style={Object {}}
>
  <p
    className="bd-notification is-success"
  >
    <p>
      Default
    </p>
    <p>
      Container
    </p>
  </p>
</div>
`;

exports[`Container component Should have helper classnames 1`] = `
<div
  className="container is-paddingless is-block-widescreen-only is-flex-touch is-hidden-desktop"
  style={Object {}}
>
  <p
    className="bd-notification is-success"
  >
    <p>
      Default
    </p>
    <p>
      Container
    </p>
  </p>
</div>
`;
